### PR TITLE
background color changed

### PR DESCRIPTION
**Pull request by tsopsericado**
I  have change the background color of the header that was initially green to antique-white.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -4,7 +4,7 @@
     font-family:'poppins', sans-serif;
 }
     #banner{
-        background: linear-gradient(rgba(0,0,0,0.5),#009688), url(./Barber_Shop_img/banner.jpg);
+      background-color: antiquewhite;
```
Comment (Gmarvis) [inline style.css]: this is wrong, it should be a background image.

**Review by Gmarvis — CHANGES_REQUESTED** [review]

File: style.css (hunk 1 above)
Comment (tsopsericado) [reply style.css]: GOOD JOB

**Comment by tsopsericado** [comment]
good job

File: style.css [hunk 2]
```diff
@@ -195,4 +195,11 @@ nav ul li a{
...
 
  
-      
\ No newline at end of file
+ /*--service--*/  
+
+ #serice{
+    width: 100%;
+    padding: 70px 0;
+    background: #efefef;
```
Comment (Gmarvis) [inline style.css:203]: ok cool we can work with this

**Review by Gmarvis — APPROVED** [review]

**Pull request merged**